ocaml/xapi-blobstore/test/dune

```diff
@@ -0,0 +1,24 @@
+(executable
+ (name xapi_blobstore_test)
+ (libraries
+   xapi_blobstore_core
+   alcotest
+   alcotest-lwt
+   threads.posix
+   logs
+   logs.fmt
+   logs.lwt
+   uuidm
+   qcheck-stm.stm
+   qcheck-stm.sequential
+   qcheck-lin.lin
+   (select spec_concurrent.ml from
+    (qcheck-stm.domain qcheck-lin.domain -> spec_concurrent.domain.ml)
+    (qcheck-stm.thread qcheck-lin.thread -> spec_concurrent.thread.ml)
+   )
+ )
+)
+
+(rule
+ (alias runtest)
+ (action (run ./xapi_blobstore_test.exe)))
```

Comment:
```suggestion
```

Reply:
fixed in a55dda750acb1428d04911cac3ac092a307f5ee9